packages/zowe-explorer/src/configuration/Definitions.ts

```diff
@@ -67,6 +67,7 @@ export namespace Definitions {
         session?: imperative.Session;
         profile?: imperative.IProfileLoaded;
         contextOverride?: string;
+        contextValue?: string;
```

Comment:
Do we need to add `contextValue` here? It feels redundant since it's already defined on tree nodes, and the `IZoweTreeOpts` type (which is only used in constructors) has `contextOverride` to support overriding the default context value.